ros-kinetic/moveit_core/moveit_core-0.9.11.ebuild

```diff
@@ -30,6 +31,11 @@ RDEPEND="
 	ros-kinetic/trajectory_msgs
 	ros-kinetic/urdf
 	ros-kinetic/visualization_msgs
+	test? ros-kinetic/angles
+	test? ros-kinetic/moveit_resources
```

Comment:
@jonbinney Got pulled in now. I didn't realize that `<test_depend>` was even a tag...